salt/modules/mine.py

```diff
@@ -41,7 +42,11 @@ def _mine_send(load, opts):
         load['tok'] = _auth().gen_token('salt')
 
     eventer = salt.utils.event.MinionEvent(opts)
-    return eventer.fire_event(load, '_minion_mine')
+    event_ret = eventer.fire_event(load, '_minion_mine')
+    # We need to pause here to allow for the decoupled nature of
+    # events time to allow the mine to propogate 
```

Comment:
I would add to the comment here that we are doing this to improve the user experience to ensure that mine updates are sequential in nature